contrib/ctr-remote/commands/rpull.go

```diff
@@ -0,0 +1,148 @@
+/*
+   Copyright The containerd Authors.
+
```

Comment:
Is keeping license header here suitable?

Reply:
I feel it's OK:)

Reply:
As the code is refer to stargz mostly，so keep the license header here